src/types/context.ts

```diff
@@ -3,13 +3,14 @@ import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as Webhook
 import { Env } from "./env";
 import { PluginSettings } from "./plugin-inputs";
 import { Logs } from "@ubiquity-dao/ubiquibot-logger";
+import { createAdapters } from "../adapters";
 
 /**
  * Update `manifest.json` with any events you want to support like so:
  *
  * ubiquity:listeners: ["issue_comment.created", ...]
  */
-export type SupportedEventsU = "issue_comment.created";
+export type SupportedEventsU = "issue_comment.created" | "issue_comment.deleted" | "issue_comment.edited";
```

Comment:
@Keyrxng can't this type be dynamically generated by importing the manifest.json file?

Reply:
Unfortunately not without trickery because string arrays type as `string` and inference goes out of the window. 

The easiest way to do it would be to convert them into keys inside the manifest as opposed to a string array this way we could easily extract string literal types from the object.